example/lib/screens/otp_screen.dart

```diff
@@ -20,21 +25,20 @@ class _OtpScreenState extends State<OtpScreen> {
 
   ok() async {
     if (myController.text.isNotEmpty) {
-      OneginiCustomTwoStepRegistrationCallback()
-          .returnSuccess(myController.text ?? " ")
-          .catchError((error) => {
-                if (error is PlatformException)
-                  {
-                    Fluttertoast.showToast(
-                        msg: error.message,
-                        toastLength: Toast.LENGTH_SHORT,
-                        gravity: ToastGravity.BOTTOM,
-                        timeInSecForIosWeb: 1,
-                        backgroundColor: Colors.black38,
-                        textColor: Colors.white,
-                        fontSize: 16.0)
-                  }
-              });
+      OneginiCustomRegistrationCallback()
+        .submitSuccessAction(widget.providerId, myController.text ?? " ")
+        .catchError((error) => {
+          if (error is PlatformException) {
+            Fluttertoast.showToast(
```

Comment:
If you reuse the same toast everywhere, you can possibly extract it with all those parameters and use it as a one liner.

Reply:
done